app/src/main/java/com/sunasterisk/weather/utils/WeatherUtils.kt

```diff
@@ -0,0 +1,66 @@
+package com.sunasterisk.weather.utils
+
+import kotlin.math.roundToInt
+import com.sunasterisk.weather.data.model.element.Direction
+import com.sunasterisk.weather.data.model.element.SpeedUnit
+import com.sunasterisk.weather.data.model.element.TemperatureUnit
+
+object WeatherUtils {
+
+    fun formatTemperature(vararg ints: Int?): String? {
```

Comment:
Đặt tên tham số ý nghĩa hơn 
```suggestion
    fun formatTemperature(vararg something: Int?): String? {
```